### DISCUSSION TITLE
dagelijks-zwitserland

### DISCUSSION THREAD

**Discussion by giscus** [Roald87/roaldin.ch#104] [Announcements]
# dagelijks-zwitserland

Een paar weken geleden heb ik de titel van dit blog verkort van “Dagelijkse dingen in Zwitserland vanuit Nederlands perspectief” naar “Zwitserse gewoonten door Nederlandse ogen”1. Laatst werd ik even herinnerd aan de oude titel, toen ik deze vrachtwagen zag vanuit de trein. “Dagelijks Zwitserland”, staat er groot op de zijkant. Ik ben benieuwd wat er dagelijks vervoerd moet worden tussen beide landen. Kastomaten misschien? https://github.com/Roald87/roaldin.ch/commit/8c08191c310705ac8240b0ef16854adfe6ade974 ↩

https://roaldin.ch/dagelijks-zwitserland

**Comment by henk-ruiter** [comment]
Kastomaten zeker niet! Die worden niet in opleggers met zeilen vervoerd.
